USART/Core/Src/main.c

```diff
@@ -1,12 +1,15 @@
 #include "stm32l053xx.h"
 #include "stdio.h"
 #include "stdarg.h"
+#include "string.h"
+
```

Comment:
Are we using any functions from string.h and stdarg.h?

Reply:
oh, no. Those are from other attempts. Gonna remove

Reply:
ok